tests/test_plugin.py

```diff
@@ -57,7 +72,7 @@ def test_a_failed_test_of_a_feature():
         """)
 
         result = testdir.runpytest('--testdox')
-        result.stdout.fnmatch_lines('module name')
+        result.stdout.fnmatch_lines(b'module name')
```

Comment:
I think the correct parameter to `fnmatch_lines` is a [list](https://docs.pytest.org/en/latest/writing_plugins.html#_pytest.pytester.LineMatcher.fnmatch_lines) (even though the provided example uses a multiline string). Using `b` is breaking the CI in python 3.x.

I tested with `result.stdout.fnmatch_lines(['module name'])` and it worked both on python 2 and python 3.